content/recipes/nestia.md

```diff
@@ -0,0 +1,243 @@
+### Nestia
+
+A set of helper libraries for NestJS, supporting the below features:
+
+  - `@nestia/core` - 15,000x faster validation decorators using [typia](https://github.com/samchon/typia)
+  - `@nestia/sdk` - SDK and Swagger Documents generator for `@nestia/core`
+  - `nestia` - just CLI tool
+
+> info **info** `nestia` is a third party package and is not managed by the NestJS core team. Please, report any issues found with the library in the [appropriate repository](https://github.com/samchon/nestia).
+
+
+
+
+#### Setup
+
+##### Boilerplate Project
+
+```bash
+npx nestia start <directory>
+```
+
+Just run above command, then boilerplate project would be constructed.
+
+##### Setup Wizard
+
+```bash
+npx nestia setup
+```
+
+When you want to use `nestia` in ordinary project, just type above command.
+
+All installation and configuration processes would be automatically done.
+
+Also, you can specify package manager or target `tsconfig.json` file like below:
+
+```bash
+npx nestia setup --manager npm
+npx nestia setup --manager pnpm
+npx nestia setup --manager yarn
+
+npx nestia setup --project tsconfig.json
+npx nestia setup --project tsconfig.test.json
+```
+
+After the setup, you can compile `@nestia/core` utilization code by using `ttsc` ([ttypescript](https://github.com/cevek/ttypescript)) command. If you want to run your TypeScript file directly through `ts-node`, add `-C ttypescript` argument like below:
+
+```bash
+# COMPILE THROUGH TTYPESCRIPT
+npx ttsc
+
+# RUN TS-NODE WITH TTYPESCRIPT
+npx ts-node -C ttypescript src/index.ts
+```
```

Comment:
Could you add an info or hint block under this explicitly pointing out that this is `ttypescript` and why it's necessary over regular `typescript` (I assume it has to do with plugins that ttypescript allows, but not 100% sure myself :)

Reply:
Fixed following your advise.